my_game.py

```diff
@@ -47,7 +47,7 @@
 PLAYER_SIGHT_RANGE = SCREEN_WIDTH/4 # How far can the player see?
 
 # Amount of players
-NUM_OF_PLAYERS = 2
+NUM_OF_PLAYERS = 1
```

Comment:
```suggestion
NUM_OF_PLAYERS = 2
```

Reply:
This is neccesary for debugging, since enemies don't work with multiplayer. Will be reverted tho